fuel-core-interfaces/Cargo.toml

```diff
@@ -13,10 +13,11 @@ description = "Fuel core interfaces"
 [dependencies]
 anyhow = "1.0"
 async-trait = "0.1"
-chrono = { version = "0.4", features = ["serde"] }
+chrono = { version = "0.4" }
```

Comment:
why was "serde" feature removed? I think it was necessary for "FuelBlockHeader" to be able to be correctly serialize `pub time: DateTime<Utc>`

Reply:
It wasn't removed, I moved it to the `serde` feature below so that we don't pull in serde when default features are disabled.

Reply:
ah good point